LICENSE

```diff
@@ -1,28 +1,28 @@
-Copyright (C) 2021 Intel Corporation
- 
-Redistribution and use in source and binary forms, with or without modification,
-are permitted provided that the following conditions are met:
- 
-1. Redistributions of source code must retain the above copyright notice,
-   this list of conditions and the following disclaimer.
+BSD 3-Clause License
+
+Copyright (C) 2021-2023 Intel Corporation
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice, this
+   list of conditions and the following disclaimer.
+
 2. Redistributions in binary form must reproduce the above copyright notice,
    this list of conditions and the following disclaimer in the documentation
    and/or other materials provided with the distribution.
-3. Neither the name of the copyright holder nor the names of its contributors
-   may be used to endorse or promote products derived from this software
-   without specific prior written permission.
- 
+
+3. Neither the name of the copyright holder nor the names of its
+   contributors may be used to endorse or promote products derived from
+   this software without specific prior written permission.
+
 THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
-THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS
-BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
-OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
-OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
-OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
-WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
-OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
-EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- 
- 
-SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Can we keep the SPDX identifier and still be automatically detected?

Reply:
Tried to place the id in different places - doesn't work. I also tried to remove only the identifier from the original version of the license - and it was recognized. The GitHub documentation has [recommendations](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#detecting-a-license:~:text=If%20your%20repository%20is%20using%20a%20license%20that%20is,complexity%20somewhere%20else%2C%20such%20as%20your%20repository%27s%20README%20file.). It will be necessary to decide whether license detection is needed or not.

Reply:
I don't think we need to keep the SPDX identifier, especially if it's making it harder to automatically detect the license.  Thanks for looking into it!